dev-docs/RFCs/v7.x/binary-data-rfc.md

```diff
@@ -21,15 +21,103 @@ This article focused on the following use cases:
 * **Using typed arrays as layer input data** - Using The input data is in binary form, perhaps it is delivered this way from the back-end, and it would be preferable to not have to unpack it.
 * **Using typed arrays or GPU buffers directly as layer attributes** - The input data is already formatted in the memory format expected by the GPU and deck.gl's shaders.
 
-## Using Flattend Data (Typed Arrays as layer input data
+## Using Flattend Data (Typed Arrays as layer input data)
 
-In some cases the application may receive all or parts of a layer's geometry in the form of binary typed arrays instead of Javascript arrays. Typed arrays can sometimes be transported more efficiently in binary form than classic arrays that tend to be JSON stringified and then parsed.
+In some cases the application may receive all or parts of a layer's geometry in the form of binary typed arrays instead of Javascript arrays. When communicating with a server or web workers, typed arrays can be transported more efficiently in binary form than classic arrays that tend to be JSON stringified and then parsed. For example, a ScatterplotLayer may receive input data like this:
```

Comment:
Maybe add a sentence to the beginning:

deck.gl layers are designed to accept an array of input data with each successive element being an object, conceptually representing a "data table row" corresponding to one visual element in. However, in some cases ...

Reply:
A similar sentence exists in the paragraph before this.